game/Enums.h

```diff
@@ -78,24 +78,24 @@ enum MoraleLevels
 //		 if the weapon is changed to ESG, EMP, Stickyfoam
 enum UnitClassifactions			// [Note: **Typo** in name]
 {
-	clsAttack = 0,				// 0 (Lynx, Panther, Tiger, Scorpion)  [Not ESG, EMP, or Stickyfoam]
-	clsESG,						// 1 (Lynx, Panther, Tiger)
-	clsEMP,						// 2 (Lynx, Panther, Tiger)
-	clsStickyfoam,				// 3 (Lynx, Panther, Tiger)
-	clsSpider,					// 4
-	clsConvec,					// 5
-	clsRepairVehicle,			// 6
-	clsCargoTruck,				// 7
-	clsEarthworker,				// 8
-	clsColony,					// 9 (clsVehicle, not specified elsewhere) (RoboSurveyor, RoboMiner, GeoCon, Scout, RoboDozer, EvacuationTransport)
-	clsVehicleFactory,			// A
-	clsArachnidFactory,			// B
-	clsStructureFactory,		// C
-	clsOreMine,					// D (CommonOreMine, RareOreMine)
-	clsGuardPost,				// E
-	clsBuilding,				// F (more like non vehicle, and non other specified class) (does not include Arachnid Factory, includes beacons, disasters, Blast, Tube, pretty much any non vehicle?)
-	clsNotSet = 0x10,			// 10
-	clsAll = 0x11,				// 11 All vehicles and buildings
+	clsAttack = 0,				// 0x0 (Lynx, Panther, Tiger, Scorpion)  [Not ESG, EMP, or Stickyfoam]
+	clsESG,						// 0x1 (Lynx, Panther, Tiger)
+	clsEMP,						// 0x2 (Lynx, Panther, Tiger)
+	clsStickyfoam,				// 0x3 (Lynx, Panther, Tiger)
+	clsSpider,					// 0x4
+	clsConvec,					// 0x5
+	clsRepairVehicle,			// 0x6
+	clsCargoTruck,				// 0x7
+	clsEarthworker,				// 0x8
+	clsColony,					// 0x9 (clsVehicle, not specified elsewhere) (RoboSurveyor, RoboMiner, GeoCon, Scout, RoboDozer, EvacuationTransport)
+	clsVehicleFactory,			// 0xA
+	clsArachnidFactory,			// 0xB
+	clsStructureFactory,		// 0xC
+	clsOreMine,					// 0xD (CommonOreMine, RareOreMine)
+	clsGuardPost,				// 0xE
+	clsBuilding,				// 0xF (more like non vehicle, and non other specified class) (does not include Arachnid Factory, includes beacons, disasters, Blast, Tube, pretty much any non vehicle?)
+	clsNotSet = 0x10,			// 0x10
+	clsAll = 0x11,				// 0x11 All vehicles and buildings
```

Comment:
Some entries already have the hex annotation as active code. Maybe we should prefer that? At least then the annotation is guaranteed to be accurate as to what the compiler does.

Reply:
Do you mean that we should specifically set the value of each enum item as:

```C++
clsAttack = 0x0,
clsESG = 0x1,
clsEMP = 0x2
```

Or that we we should just standize the comments to be hexadecimal?

Reply:
Yes, I believe we should set the value in code, rather than annotated as a comment. That way the value is guaranteed to be what is stated.